### PR TITLE
Cleanup blueprints

### PR DESCRIPTION
**Pull request by normj**
*Description of changes:*
Clean up a few miscellaneous issues with the blueprints.

* Update XUnit NuGet references
* Switch Annotations blueprint to use filescoped namespaces
* Removed stale documentation annotations that was useful before .NET 6 managed runtime shipped
* Improve NDoc for ASP.NET Core minimal API blueprint
* Fix S3 blueprint to not return a value which is was not a valid use case for S3 event function
* Convert S3 blueprint test to be a unit test.
* Fixed up issues where the source file would be called `Function` but the class would be `Functions`.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: Blueprints/BlueprintDefinitions/vs2022/AnnotationsFramework/template/test/BlueprintBaseName.1.Tests/FunctionTest.cs [hunk 1]
```diff
@@ -1,21 +1,20 @@
 using Xunit;
 using Amazon.Lambda.TestUtilities;
 
-namespace BlueprintBaseName._1.Tests
+namespace BlueprintBaseName._1.Tests;
+
+public class FunctionTest
```
Comment (96malhar) [inline Blueprints/BlueprintDefinitions/vs2022/AnnotationsFramework/template/test/BlueprintBaseName.1.Tests/FunctionTest.cs:6]: The class name and the file name should be changed to `FunctionsTest`

**Review by 96malhar — CHANGES_REQUESTED** [review]

File: Blueprints/BlueprintDefinitions/vs2022/SimpleS3Function/template/test/BlueprintBaseName.1.Tests/FunctionTest.cs [hunk 2]
```diff
@@ -1,65 +1,52 @@
...
 using Amazon.S3.Model;
-using Amazon.S3.Util;
-using System.Collections.Generic;
+using Moq;
+using Xunit;
 
 namespace BlueprintBaseName._1.Tests;
 
-public class FunctionTest
+public class FunctionsTest
```
[the remark this answers is not in the document]
Comment (normj) [reply Blueprints/BlueprintDefinitions/vs2022/SimpleS3Function/template/test/BlueprintBaseName.1.Tests/FunctionTest.cs]: Fixed

File: Blueprints/BlueprintDefinitions/vs2022/SimpleS3Function/template/src/BlueprintBaseName.1/Function.cs [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
 namespace BlueprintBaseName._1;
 
-public class Function
+public class Functions
```
[the remark this answers is not in the document]
Comment (normj) [reply Blueprints/BlueprintDefinitions/vs2022/SimpleS3Function/template/src/BlueprintBaseName.1/Function.cs:11]: Fixed

File: Blueprints/BlueprintDefinitions/vs2022/AnnotationsFramework/template/test/BlueprintBaseName.1.Tests/FunctionTest.cs (hunk 1 above)
Comment (normj) [reply Blueprints/BlueprintDefinitions/vs2022/AnnotationsFramework/template/test/BlueprintBaseName.1.Tests/FunctionTest.cs]: Fixed

**Review by 96malhar — APPROVED** [review]

**Pull request merged**